### DISCUSSION TITLE
Resource utilization

### DISCUSSION THREAD

**Discussion by harrishere** [NVlabs/sionna#245] [General]
Hi!

It's a question about resource utilization. The 2x1 scenario seems to have more average GPU utilization and memory utilization as compared to the 4x2 case.

Whereas, 4x2 seems to have the maximum GPU utilization and memory utilization as compared to 2x1.

But if we compare only the average utilization, then why does the 4x2 case have less resource utilization if we compare it with the 2x1 case?

It seems like the 4x2 case is more optimized as compared to the 2x1?

Best, Haris.

**Comment by jhoydis** [comment]
Hi,

Can you share a little bit of background to this question, ideally some example code?

**Reply by harrishere** [reply]
Hi! The problem was solved. Thank you!
